client/src/components/UsedOnTable/provideUsedOnData.js

```diff
@@ -2,26 +2,97 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { loadUsedOn } from 'state/usedOn/usedOnActions';
+// import { injectTabContext } from 'hooks/useTabContext'; TODO uncomment
```

Comment:
There's a bunch of TODO's in this file to uncomment/remove bits of code

This is specifically so that it can
1) work out of the box right now (along with this [PR](https://github.com/silverstripe/silverstripe-asset-admin/pull/1139))
2) after doing all of the TODO's, work in the future state once the [tab context PR](https://github.com/silverstripe/silverstripe-admin/pull/1129) is merged